README.md

```diff
@@ -114,22 +114,11 @@ If you are starting your journey with `js-libp2p`, read the [GETTING\_STARTED.md
 
 ### Tutorials and Examples
 
-You can find multiple examples on the [examples folder](./examples) that will guide you through using libp2p for several scenarios.
+You can find multiple examples on the [examples repo](https://github.com/libp2p/js-libp2p-examples) that will guide you through using libp2p for several scenarios.
```

Comment:
i feel like there should be other tutorials we could link to here, but i cant think of any other than protoschool which needs updating

Reply:
> Pours one out for Protoschool.

😢